flink-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/JDBCLookupFunction.java

```diff
@@ -76,7 +87,8 @@
 	private final int maxRetryTimes;
 
 	private transient Connection dbConn;
-	private transient PreparedStatement statement;
+	private transient PreparedStatement fastStatement;
+	private transient PreparedStatement slowStatement;
```

Comment:
If we have a consensus on the new methods of `SqlDialect`. We can rename these fields to `nullSafeStatemen` and `nullUnsafeStatement`, we can add comment on the fields that we should use `nullUnsafeStatement` as much as possible because it is faster.